server/neptune/workflows/internal/deploy/workflow.go

```diff
@@ -76,6 +76,9 @@ func newRunner(ctx workflow.Context, request Request, terraformWorkflow func(ctx
 			InstallationToken: request.Repository.Credentials.InstallationToken,
 		},
 	}
+
+	// TODO: We should actually probably pass this with the revision because a revision
+	// can potentially change a root configuration
```

Comment:
any reason to not pass the revision in with the repo/root we forward with the DeployRequest?